src/js/first_parties/twitter.js

```diff
@@ -0,0 +1,16 @@
+function unwrapTwitterURLs() {
+  let aElems = document.querySelectorAll("a[data-expanded-url][href^='https://t.co/']");
+  let n = aElems.length;
+  for (let i = 0; i < n; i++) {
+    let elem = aElems[i];
+    let deu = elem.getAttribute("data-expanded-url");
+    if (deu && (deu.startsWith("https://") || deu.startsWith("http://"))) {
+      elem.href = deu;
+      elem.addEventListener("click", function (e) {
+        e.stopPropagation(); // don't let the event click propogate to twitter
+      });
+    }
+  }
+}
+
+setInterval(unwrapTwitterURLs, 5000);
```

Comment:
We probably want to call `unwrapTwitterURLs` right away, right? But `setInterval` delays first execution too. Also, generally speaking (maybe not so much here), `setInterval` is risky since it's not guaranteed to execute on time or evenly spaced or anything so it can lead to your callbacks piling up. Safer to repeat by having the function call itself with `setTimeout` at the end.